mobile/native/src/ln_dlc/mod.rs

```diff
@@ -544,13 +537,23 @@ fn derive_trades_from_filled_orders() -> Result<Vec<WalletHistoryItem>> {
                 .trader_margin()
                 .expect("Filled order to have a margin");
 
+            let execution_price = Decimal::try_from(
+                first
+                    .execution_price()
+                    .context("execution price to be set on a filled order")?,
+            )?;
+            let fee = order_matching_fee_taker(first.quantity, execution_price).to_sat();
+            let amount_sats = amount_sats + fee;
```

Comment:
🔧 I think we should leave a comment here as this is not intuitive.

If I understand it correctly, we want to convey that the amount spent in this trade is the sum of the margin + the opening fee.